services/QuillLMS/spec/models/feedback_history_spec.rb

```diff
@@ -640,5 +643,41 @@
         expect(@feedback_history.most_recent_rating).to eq true
       end
     end
+
+    context '#session_data_for_csv' do
+      it 'should take the query from #session_data_for_csv and return a shaped payload' do
```

Comment:
Cool.  I haven't seen this `RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length ` setting before. 

One to keep in mind with changing defaults in RSpec is you want to make sure that you set them back to the original.  Otherwise, they can carry on into other specs.  (this is similar to why we use `stub_const` in before blocks).  

```
      around do |example|
        default_length = RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length
        RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length = 10000
        example.run
        RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length = default_length
      end

      it 'should take the query from #session_data_for_csv and return a shaped payload' do
      ...
```

Reply:
@brendanshean thanks for the explanation! I actually took this from the existing `#serialize_list_by_activity_session` spec, but I'm wondering what a good default length would be? I'm looking through the docs, but not finding anything eluding to what the default is under the hood